conjure-codegen/src/errors.rs

```diff
@@ -18,12 +18,18 @@ use crate::context::Context;
 use crate::objects;
 use crate::types::{ErrorDefinition, ObjectDefinition};
 
+impl ErrorDefinition {
+    pub fn object_definition(&self) -> ObjectDefinition {
```

Comment:
Can we instead make this a free function? I try to avoid adding new impls on generated types.

Reply:
done, though lemme know if you have a preferred naming convention

Reply:
LGTM!